### PR TITLE
Add libvirt provider for vagrant

### PR DESCRIPTION
**Pull request by pronix**
additional way to avoid virtualbox and speedup vm.

### CODE REVIEW

**Comment by derekwaynecarr** [comment]
This is cool.  I will look to try this out tomorrow time permitting.  Does this allow for parallel boot of the boxes?  Can you clarify which version of vagrant-libvirt plugin you tested this on and on what host OS?

**Comment by pronix** [comment]
hello
yes, allow parallel, vagrant 1.7.2, vagrant-libvirt 0.0.26, os - fedora21

File: Vagrantfile [hunk 1]
```diff
@@ -123,6 +130,15 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       v.vmx['numvcpus'] = $vm_cpus
     end
 
+    # configure libvirt provider
+    config.vm.provider :libvirt do |v, override|
+      setvmboxandurl(override, :libvirt)
+      v.memory = vm_mem
+      v,cpus = $vm_cpus
```
Comment (Fodoj) [inline Vagrantfile]: that should be a dot rather than comma, or?..

Comment (pronix) [inline Vagrantfile]: thanks.fixed.

**Comment by derekwaynecarr** [comment]
Can you squash the commits?

Are you able to assist with any libvirt specific issues in the future?

**Comment by derekwaynecarr** [comment]
Can you update the document here to include information on the prerequisite versions of plugins, etc.?

https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/getting-started-guides/vagrant.md

**Comment by pronix** [comment]
Hello.
Today will squashed and added information to docs.
Thanks

**Comment by pronix** [comment]
hello
squashed, added short description and link to docs.
And sure I will support with any question

**Comment by derekwaynecarr** [comment]
LGTM, thanks a lot for your contribution.

**Pull request merged**